utils/deploy-aws/README.md

```diff
@@ -11,6 +11,13 @@ other tests can be performed.
 Getting Started
 ---------------
 
+### Prerequisites
+
+1. Terraform: download and install [here](https://www.terraform.io/downloads.html).
+2. Ansible: on recent Ubuntu, simply use:
+
```

Comment:
```suggestion
2. Ansible: on recent Ubuntu, simply use:

```

Somehow it doesn't hightlight without extra newline.

Reply:
Done